Tasks/NodeTool/nodetool.ts

```diff
@@ -103,7 +104,11 @@ async function getNode(versionSpec: string, onlyLTS: boolean) {
             // get the latest version that matches the version spec
             //
             version = toolLib.evaluateVersions(versions, versionSpec);
-            toolLib.debug('version from index.json', version);
+            if (!version) {
```

Comment:
with the current logic, version ended up being undefined so it would never work. so i added this validation to avoid weird error later.